src/app/passengers/components/smart/trip-planner/trip-planner.component.ts

```diff
@@ -304,12 +278,11 @@ export class TripPlannerComponent implements OnInit {
       withCredentials: true,
       responseType: 'text' as 'text'
     };
-    
 
     const inputElem = JSON.stringify({
       'origin': this.pickupInputElementRef.nativeElement.value,
       'destination': this.pickupOutputElementRef.nativeElement.value,
-      'userId': '560c62f4-8612-11e8-adc0-fa7ae01bbebc'
+      'userId': localStorage.getItem('userId')
```

Comment:
Can we just ask the authentication service for this value via a new function, so we aren't leaking the implementation details?

Reply:
Yes good suggestion